examples/source-maps/rollup.config.js

```diff
@@ -0,0 +1,35 @@
+const { terser } = require('rollup-plugin-terser')
+const { copyFileSync } = require('fs')
+
+const after = () => {
+  return {
+    generateBundle (outputOptions, bundle, isWrite) {
+      setTimeout(() => {
+        copyFileSync('./index1.min.js.map', './index2.min-map.js')
+      }, 1)
```

Comment:
What is this timeout for?

Reply:
Thats probably not required anymore, at first I also create a source map for index2 and then renamed that one to one that couldnt be resolved automatically. But this didnt work well because the generated files didnt exist yet in this hook, they did in nextTick (which I implement here with the timeout). I can check if I can remove this now